lib/apress/selenium_products/spec/company_site/kirby/import/import_yml_policy_spec.rb

```diff
@@ -0,0 +1,434 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'Кирби: политики доступа страницы импорта YML', feature: 'Кирби: политики доступа страницы импорта YML' do
+  before(:all) do
+    @import_yml_page = CompanySite::ImportYMLPage.new
+    @upload_tab      = CompanySite::ImportYMLPage::UploadTab.new
+    @settings_tab    = CompanySite::ImportYMLPage::SettingsTab.new
+  end
+
+  shared_examples 'общие элементы страницы' do
+    it 'присутствует title в формате "Загрузка товаров из YML/YRL - %название компании% - %основной регион%"' do
+      expect(@import_yml_page.has_expected_title?).to be true
+    end
+
+    it 'присутствует заголовок страницы "Загрузить товары в формате Яндекс.Маркет (YML/YRL)"' do
+      expect(@import_yml_page.header).to eq 'Загрузить товары в формате Яндекс.Маркет (YML/YRL)'
+    end
+
+    it 'присутствуют вкладки "Загрузить товары" и "Расширенные настройки"' do
+      expect(@import_yml_page.upload_tab?).to be true
+      expect(@import_yml_page.settings_tab?).to be true
+    end
+  end
+
+  shared_examples 'все элементы страницы для роли с повышенными правами' do |role|
+    context 'когда на вкладке "Загрузить товары"' do
+      before(:all) { @import_yml_page.upload_tab }
+
+      it('присутствует текстовая зона') { expect(@upload_tab.text_zone?).to be true }
+
+      it 'присутствует активный блок "Формат" со значениями YML, YRL и EDIFACT' do
+        expect(@upload_tab.source_type_label?).to be true
+        expect(@upload_tab.source_type_options).to eq %w[YML YRL EDIFACT]
+      end
+
+      it('доступен режим "Товаров нет"') { expect(@upload_tab.moving_no?).to be true }
+
+      context 'когда выбран режим "Товары есть"' do
+        before(:all) { @upload_tab.moving_yes }
+
+        it('доступен режим "Оставить опубликованными"') { expect(@upload_tab.moving_keep?(2)).to be true }
+        it('доступен режим "Удалить"') { expect(@upload_tab.moving_delete?(2)).to be true }
+        it('доступен режим "Перенести в архив"') { expect(@upload_tab.moving_archive?(2)).to be true }
+
+        it('присутствует кнопка "Загрузить"') { expect(@upload_tab.upload_price?).to be true }
+      end
+
+      #TODO: Раскипать после того, как обновят образ ножа для ПЦ, тест пропускается так как файла по пути /test_files/pulscen пока нет.
```

Comment:
[Correctable] Layout/LeadingCommentSpace: Missing space after #.<br>Layout/LineLength: Line is too long. [135/120]